code/__defines/damage_organs.dm

```diff
@@ -64,6 +64,16 @@
 #define BLOOD_VOLUME_BAD     60
 #define BLOOD_VOLUME_SURVIVE 30
 
+
+// Oxyginetion states
+#define OXYGENATION_STATE_HIGH 5
+#define OXYGENATION_STATE_NORMAL 4
+#define OXYGENATION_STATE_LOW 3
+#define OXYGENATION_STATE_VERY_LOW 2
+#define OXYGENATION_STATE_NONE 1
+#define OXYGENATION_STATE_UNDEFINED 0
```

Comment:
I should have perhaps been a bit more specific, but global defines for this are also bad. As with variables, the scope of macros should be limited (you wouldn't declare a global var when a local will suffice), so these should be `#define`-d at the top of the `code/datums/repositories/crew.dm` file and `#undef`-d at the bottom of the same file.

`__defines` is for project-wide (global) defines, not _all_ defines.